paella-matterhorn/plugins/edu.harvard.dce.infoButtonPlugin/dce_infobutton.js

```diff
@@ -99,7 +117,36 @@ Class ('paella.plugins.InfoPlugin', paella.ButtonPlugin,{
 
   getVideoUrl: function () {
     return document.location.href;
+  },
+
+  checkItemEnabled: function(item) {
+    if (item === this._classHandoutKey) {
+       var isenabled = this.checkClassHandouts();
+       return isenabled;
+     }  else {
+       return true;
+     }
+  },
+
+  checkClassHandouts: function () {
```

Comment:
Handouts is not a separate plugin anymore. The Handout  link is controlled by the info plugin.